### PR TITLE
Implement endpoint to add experience and return the index of the experience closes #22 #4

### LINKED ISSUE #4: Delete existing Education
Using a `DELETE` request for the `/resume/education` route, delete an existing Education using its index position as an ID. As part of this, you should write a test in `test_pytest.py` to show it works.

### LINKED ISSUE #22: Implement API route to add new "Experience"
On the `/resume/experience` route, you should:

- [ ] Return the index position of the new Experience in the Experience list when a POST request is made

### PR DESCRIPTION
**Pull request by asuzukosi**
Add method to add experience to json object and return the index of the experience once added

Closes #22 
Closes #4

### CODE REVIEW

File: .gitignore
Comment (Pradyuman7) [inline .gitignore:1]: Do we actually need all these directories in the .gitignore?

File: models.py [hunk 1]
```diff
@@ -7,6 +7,15 @@
 from dataclasses import dataclass
 
 
+@dataclass
+class PersonalInfo:
+    '''
+    User details
+    '''
+    first_name: str
```
Comment (Pradyuman7) [inline models.py:15]: nit: you could also have a optional middle_name

File: models.py [hunk 2]
```diff
@@ -41,3 +50,14 @@ class Skill:
...
     logo: str
+
+
+@dataclass
+class Project:
+    '''
+    Project class
+    '''
+    name: str
+    language: str
```
Comment (Pradyuman7) [inline models.py:61]: I'd say that multiple languages for a project might be better, as it'd support adding projects with multiple languages easily.

**Review by Pradyuman7 — CHANGES_REQUESTED** [review]
Please make sure that the tests and the build is passing

File: .gitignore
Comment (asuzukosi) [reply .gitignore:1]: Okay would remove the unnecessary ones

File: models.py (hunk 1 above)
Comment (asuzukosi) [reply models.py:15]: Sure, would add

**Comment by asuzukosi** [comment]
@Pradyuman7 The build tests are other peoples tasks, and I don't want to repeat their work

**Review by Pradyuman7 — APPROVED** [review]
LGTM 💯

**Comment by asuzukosi** [comment]
Thanks @Pradyuman7 !

**Pull request merged**